airflow/www/views.py

```diff
@@ -394,8 +394,10 @@ def health(self):
         return wwwutils.json_response(payload)
 
     @expose('/home')
-    @has_access
-    def index(self):  # pylint: disable=too-many-locals,too-many-statements
+    @auth.has_access([
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_WEBSITE),
```

Comment:
So other than this one case, the website permission feels entirely needless. I wonder if this one could use `CAN_READ, RESOURCE_DAGS` here, and entirely the website permissions.

(can read dags here, because the home page is effectively a list of the DAGs)